news/_posts/2020-12-09-pr-testing-on-prestashop.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title:  "PR Testing on PrestaShop"
+subtitle: ""
+date:   2020-12-08 16:00:00
+authors: [ RobinFischer ]
+icon: icon-airplane
+tags:
+ - 
+---
+
+Prestashop created a Quality Assurance team 4 years ago, in order to improve the quality of the solution.
+For more than a year now, the team has grown quickly, allowing it to take quality to the next level ! The field of action is widening (test of new services like Prestashop Checkout, specification test and review…), but the core of the manual testers activity remains the same; testing the Pull Requests (“PR”) on Github, and testing every release.
+
+
+# I. Some questions you may have in mind
+
+### Why do you test every PR ?
+
+First question that might come to your mind : Why is there QA on each PR ? Why not testing only during build release ?
```

Comment:
I think it would be more nice to use `Pull Requests` than `PR`. `PR` can be used in a sentence where you write multiple times about it.

"PR" is rather for speaking with colleagues 😉  less for writing